api/test/wfl/system/v1_endpoint_test.clj

```diff
@@ -219,30 +219,32 @@
 
 (defn ^:private covid-workload-request []
   "Build a covid workload request."
-  (let [source   {:name    "Terra DataRepo"
-                  :dataset "79fc88f5-dcf4-48b0-8c01-615dfbc1c63a"
-                  :table   "flowcells"
-                  :column  "last_modified_date"}
-        executor {:name                       "Terra"
-                  :workspace                  "wfl-dev/CDC_Viral_Sequencing"
-                  :methodConfiguration        "cdc-covid-surveillance/sarscov2_illumina_full"
-                  :methodConfigurationVersion 1
-                  :fromSource                 "importSnapshot"}
-        sink     {:name           "Terra Workspace"
-                  :workspace      "wfl-dev/CDC_Viral_Sequencing"
-                  :entityType     "assemblies"
-                  :identifier     "flowcell_id"
-                  :fromOutputs    (resources/read-resource
-                                   "sarscov2_illumina_full/entity-from-outputs.edn")
-                  :skipValidation true}]
+  (let [terra-ns  (comp (partial str/join "/") (partial vector "wfl-dev"))
+        workspace (terra-ns "CDC_Viral_Sequencing")
```

Comment:
not sure what this adds for readability.

Reply:
Not much, if anything.  I'm open to suggestions.
Especially considering the table formatting forced a lot of whitespace changes.
`(partial str "wfl-dev/")` seemed hacky and everything else too ugly.
(-I should just stop trying to adopt your aesthetic!-)